src/wifipaf/WiFiPAFEndPoint.cpp

```diff
@@ -777,12 +767,12 @@ SequenceNumber_t WiFiPAFEndPoint::AdjustRemoteReceiveWindow(SequenceNumber_t las
                                                             SequenceNumber_t newestUnackedSentSeqNum)
 {
     // Assumption: SequenceNumber_t is uint8_t.
```

Comment:
The comment re-states the code. Could we please state *why* we have this assumption (which is not obvious from the code)? Me reading this just asks "so why not a 16 or 32 bit sequence? longer sequence support sounds great..." ... I should understand the limitation from code/comments